ragna/deploy/_api/schemas.py

```diff
@@ -66,6 +83,14 @@ def from_core(cls, message: ragna.core.Message) -> Message:
             sources=[Source.from_core(source) for source in message.sources],
         )
 
+    @classmethod
+    def to_core(cls, message: Message) -> ragna.core.Message:
+        return ragna.core.Message(
+            content=message.content,
+            role=message.role,
+            sources=[Source.to_core(source) for source in message.sources],
+        )
```

Comment:
I don't understand why these are class methods. They should be default instance methods

```suggestion
    def to_core(self) -> ragna.core.Message:
        return ragna.core.Message(
            content=self.content,
            role=self.role,
            sources=[source.to_core() for source in self.sources],
        )
```

Reply:
That makes way more sense. Idk why I did it that way.